### PR TITLE
[4.0] Namespace com_actionlogs

### PR DESCRIPTION
**Pull request by hardik-codes**
This PR namespaces com_actionlogs

### CODE REVIEW

**Comment by Quy** [comment]
Please revert removing blank lines.

**Comment by hardik-codes** [comment]
done @Quy

File: administrator/components/com_actionlogs/models/fields/extension.php [hunk 1]
```diff
@@ -9,7 +9,11 @@
 
 defined('_JEXEC') or die;
 
-JFormHelper::loadFieldClass('list');
+use Joomla\CMS\Factory;
+use Joomla\CMS\Form\FormHelper;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+FormHelper::loadFieldClass('list');
```
Comment (Quy) [inline administrator/components/com_actionlogs/models/fields/extension.php:17]:
```suggestion

FormHelper::loadFieldClass('list');
```

**Comment by hardik-codes** [comment]
Thanks @Quy

**Comment by fastslack** [comment]
@test All seems ok. Added a PR to this branch with code styling

https://github.com/hardik-codes/joomla-cms/pull/1

**Comment by hardik-codes** [comment]
@fastslack please mark your test

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cab4a7795fe479c8e940362496eebcd5c2e85f71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24764">issues.joomla.org/tracker/joomla-cms/24764</a>.</sub>

**Comment by ghost** [comment]
@fastslack how to mark your test as successfully > https://docs.joomla.org/Testing_Joomla!_patches#Recording_test_results

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on cab4a7795fe479c8e940362496eebcd5c2e85f71<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24764">issues.joomla.org/tracker/joomla-cms/24764</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24764">issues.joomla.org/tracker/joomla-cms/24764</a>.</sub>

**Comment by SniperSister** [comment]
Merged, thanks a lot for your contribution, it's highly appreciated!

**Pull request merged**